fast/stages/02-networking-vpn/data/subnets/dev/dev-dp-lod-ew1.yaml

```diff
@@ -0,0 +1,5 @@
+# skip boilerplate check
```

Comment:
More of a question than a comment: do we **need** this many subnets?
Can we reuse any?